docker/Dockerfile

```diff
@@ -1,19 +1,14 @@
+# To build this container, go to ESMValCore root folder and execute :
+# docker build -t ${TAG_OF_YOUR_CHOICE} . -f docker/Dockerfile
 FROM continuumio/miniconda3
 
-# update the conda packages
-RUN conda update -y conda pip
+COPY . /src/ESMValTool
+WORKDIR /src/ESMValTool
 
-# install development tools
-RUN apt-get update -y && apt-get install -y \
-    build-essential \
-    curl \
-    unzip
-
-# install environment packages
-RUN conda install -c conda-forge -c esmvalgroup -c birdhouse esmvaltool
-
-# run tests
-RUN esmvaltool -h
+RUN conda update -y conda pip && conda env update --name base --file environment.yml && conda clean --all -y
```

Comment:
Maybe you could COPY just environment.yml before this command? That would work better with the [build cache](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#leverage-build-cache)? And maybe move the R installation to just after the conda env update? Because that's also something you preferably don't want to do too often.

Reply:
Done.  I can manually install the R dependencies before copying the full ESMValTool, but in that case we can not use the install command.

I also added a `dockerignore` so we can avoid rebuilds when just changing the doc or the dockerfiles (at least until the changed command)